src/library/commissioner_impl.cpp

```diff
@@ -149,7 +149,9 @@ CommissionerImpl::CommissionerImpl(CommissionerHandler &aHandler, struct event_b
     , mResourceUdpRx(uri::kUdpRx, [this](const coap::Request &aRequest) { mProxyClient.HandleUdpRx(aRequest); })
     , mResourceRlyRx(uri::kRelayRx, [this](const coap::Request &aRequest) { HandleRlyRx(aRequest); })
     , mProxyClient(mEventBase, mBrClient)
+#if OT_COMM_CONFIG_CCM_ENABLE
     , mTokenManager(mEventBase)
+#endif // OT_COMM_CONFIG_CCM_ENABLE
```

Comment:
nit: can avoid these comments when block is less than 20 lines

Reply:
removed those comments.